readme_renderer/rst.py

```diff
@@ -95,7 +103,11 @@ def emptytag(self, node, tagname, suffix="\n", **attributes):
 }
 
 
-def render(raw, stream=None, **kwargs):
+def render(
+    raw: str,
+    stream: typing.Optional[typing.Any] = None,
```

Comment:
I _think_ this could be:

```suggestion
    stream: typing.Optional[typing.IO[str]] = None,
```

Via https://mypy.readthedocs.io/en/stable/cheat_sheet.html#miscellaneous